### ISSUE TITLE
Parsing Bug When Using EOF

### ISSUE THREAD

**Issue by afrazkhan** [hashicorp/vscode-terraform#176]
Uber specific parsing error that results in a false positive which annoys me enough to come and post this after 2 months ;)

When using multiline with something like `<<EOF`, the appearance of ":" in combination with "${" on any line will result in the false positive syntax error: "expected "}" but found ":""

e.g.

```
    variable "foo" {
      default = <<EOF
${bar::-1}
EOF
    }
```

Taking out the colons gets rid of the error.

**Comment by afrazkhan** [comment]
This bug is not present in version 1.3.9 (there's no tag for that version here, but it's what I have installed from VSCode's extension manager).

**Comment by mauve** [comment]
this is most certainly because we do not support terraform 0.12 yet, see #157, you can try disabling `terraform.indexing.enabled` in the settings

**Comment by afrazkhan** [comment]
I'm not on 0.12 though? I'm on 0.11.13.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
